config/eye.yml.erb

```diff
@@ -1,7 +1,6 @@
-<% app_root = File.expand_path("..", __dir__) %>
-<% rails_env = ENV["RAILS_ENV"] %>
----
-name: NUcore (<%= rails_env %>)
```

Comment:
I would acutally say that you should just keep this

Reply:
In the rails console I'm getting:
```
2.5.5 :001 > File.expand_path("..", __dir__)
=> "/home/nucore/nucore.stage.tablexi.com/releases"
```

Reply:
Maybe I misunderstood - is your comment about `rails_env`, `app_root`, or both?